### PR TITLE
Update third-party java components version.

### PR DESCRIPTION
**Pull request by osokin**

### CODE REVIEW

File: auto/modules/nodejs [hunk 1]
```diff
@@ -158,7 +158,7 @@ ${NXT_NODE_VERSION_FILE}: ${NXT_VERSION_H}
 	$echo '#define NXT_NODE_VERNUM \$(NXT_VERNUM)' > $NXT_NODE_VERSION_FILE
 
 ${NXT_NODE_TARBALL}: ${NXT_NODE}-copy
-	sed -e 's/"version"\s*:.*/"version": "\$(NXT_VERSION)",/' \
+	sed -e 's/"version"[[:space:]]*:.*/"version": "\$(NXT_VERSION)",/' \
```
Comment (VBart) [inline auto/modules/nodejs:161]: I suggest to use something like:
```bash
sed -e "s|%%VERSION%%|\$(NXT_VERSION)|"
```
and replace `1.0.0` in the `package.json` with a distinct placeholder like `%%VERSION%%`.

Comment (osokin) [reply auto/modules/nodejs:161]: That looks good to me.

File: auto/modules/java [hunk 2]
```diff
@@ -297,7 +297,7 @@ NXT_JAR_NAME=jetty-http
 . auto/modules/java_get_jar
 
 NXT_JAR_NAME=classgraph
-NXT_JAR_VERSION=4.8.90
+NXT_JAR_VERSION=4.8.98
```
Comment (mar0x) [inline auto/modules/java:300]: According to https://mvnrepository.com/artifact/io.github.classgraph/classgraph , version 4.8.102 released in January. Is there any reason to use 4.8.98 ?

Comment (osokin) [reply auto/modules/java:300]: Missed that one, please use the latest, i.e. 4.8.102.  Thanks.

**Review by 888520ll19811209ll — APPROVED** [review]

**Review by 888520ll19811209ll — APPROVED** [review]

**Pull request closed without merging**